user/pages/40.governance/10.constitution/default.en.md

```diff
@@ -97,14 +100,15 @@ Kanthaus is an open-entry common-space dedicated to the common good. It is gover
 1. A Volunteer should—
 	1. assume some responsibility for the space, and
 	1. contribute to Wurzen and/or the wider world.
-1. A Volunteer should be evaluated (§9.) as soon as possible after 2 months (60 days). At their evaluation they may request to—
-  1. return to Guest,
+1. A Volunteer should be evaluated (§6.) as soon as possible after 2 months (60 days) elapsed since their last evaluation. At their evaluation they may request to—
+  1. return to Visitor,
   1. continue as a Volunteer, or
   1. progress to Member.
-1. A Volunteer may return to Guest at any point.
+1. A Volunteer may return to Visitor at any point.
 1. A person ceases to be a Volunteer—
-	1. if they return to Guest or progress to Member,
-	1. if decided upon by Conflict Resolution by Intervention (§13c.)
+	1. if they return to Visitor or progress to Member,
+    1. if they did not spend time in the space for 3 months,
```

Comment:
better write it similar to other time frames in the constitution: `3 months (90 days)`